utils/mediaupload.js

```diff
@@ -1,45 +1,46 @@
 /**
  * External Dependencies
  */
-import { compact } from 'lodash';
+import { compact, startsWith } from 'lodash';
 
 /**
- *	Media Upload is used by image and gallery blocks to handle uploading an image.
+ *	Media Upload is used by audio, image, gallery and video blocks to handle uploading a media file
  *	when a file upload button is activated.
  *
  *	TODO: future enhancement to add an upload indicator.
  *
  * @param {Array}    filesList      List of files.
  * @param {Function} onImagesChange Function to be called each time a file or a temporary representation of the file is available.
+ * @param {string}   allowedType    The type of media that can be uploaded.
  */
-export function mediaUpload( filesList, onImagesChange ) {
+export function mediaUpload( filesList, onImagesChange, allowedType ) {
```

Comment:
Should we rename `onImagesChange` => `onChange` or `onMediaChange`

Reply:
I missed that :(

Reply:
I picked `onFileChange` to reflect the param's description.